docs/mxnet_backend/installation.md

```diff
@@ -135,30 +138,25 @@ $  nvidia-smi
 
 ### Install Keras with MXNet backend
 
-TODO: `update: change to keras-mxnet package after release`
 ```bash
-    $ pip install keras-mxnet --pre --user
+    $ pip install keras-mxnet --user
 ```
 
-# 2 Configure Keras backend
+# 2 Configure Keras backend and image_data_format
 
-You should set Keras backend to 'mxnet' to run your Keras code with MXNet backend. You can either set the environment variable or edit the keras config file.
+When you install the `keras-mxnet`, by default, the following values are set.
 
-`Option 1` Set the environment variable
-
-```bash
-    $ export KERAS_BACKEND=mxnet
+```json
+backend: mxnet
+image_data_format: channels_last
 ```
 
-`Option 2` Edit the keras config file
-
-Open "*~/.keras/keras.json*" file and set "*backend=mxnet*".
+We strongly recommend changing the image_data_format to `channels_first`. MXNet is significantly faster on 'channels_first' data. Default is set to 'channels_last' with an objective to be compatible with majority of existing users of Keras. See [performance tips guide](performance_guide.md) for more details.
```

Comment:
are we showing this message prior to model execution as well in the cli?

Reply:
We show this message when user loads keras with mxnet backend.